scripts/render-docs-for-components.sh

```diff
@@ -6,7 +6,8 @@ GITHUB_ORG=${GITHUB_ORG:-"cloudposse"}
 GITHUB_REPO=${GITHUB_REPO:-"terraform-aws-components"}
 #GIT_BRANCH=${GIT_BRANCH:-"main"}
 # until https://github.com/cloudposse/terraform-aws-components/pull/1085 is merged
-GIT_BRANCH=${GIT_BRANCH:-"add-frontmatter"}
+#GIT_BRANCH=${GIT_BRANCH:-"add-frontmatter"}
+GIT_BRANCH=${GIT_BRANCH:-"DEV-2453/admonitions"}
```

Comment:
```suggestion
GIT_BRANCH=${GIT_BRANCH:-"add-frontmatter"}
```
Once https://github.com/cloudposse/terraform-aws-components/pull/1092 is merged, update `add-frontmatter` branch and commit this suggestion